src/15utility.js

```diff
@@ -553,6 +582,14 @@ var deleteFile = utils.deleteFile = function(path,cb){
     if(utils.isNode) {
         var fs = require('fs');
         fs.unlink(path, cb);
+	} else if(utils.isReactNative) {
+        /* If ReactNative */

```

Comment:
As this is also stated in the `If` just above, I recommend ` /* If ReactNative */`gets removed